scripts/docs.js

```diff
@@ -275,12 +283,74 @@ globby('./lib/*{,/*}.js', { ignore: IGNORE }).then(function(files) {
       }
 
       var docs = parseDocs(file, contents, 0);
-      var outputFile = path.join(
-        './docs/json/master',
-        file.replace('/lib', '').replace('.js', '.json')
-      );
 
-      fs.writeFile(outputFile, JSON.stringify(docs), callback);
+      function writeFile() {
```

Comment:
Rather than lumping this all in at the end, could we just create a `spellcheck` function and reference it within the `formatHtml` function? Everything that we would want to spell check (classdesc, decription, example comments, etc.) is going to get passed through there. I think this would simplify the code a bit and allow us to avoid traversing the JSON.